INSTALLATION.md

```diff
@@ -80,6 +80,11 @@ Congratulations! You have successfully installed **Massa Station** on your compu
 5. Proceed to GUI installation starting at step 2. 
 
 
+### Linux HTTPS workaround
+1. Open your browser "Manage Certificates" settings (e.g. chrome://settings/certificates or brave://settings/certificates) and switch to the "Authorities" tab.
```

Comment:
I think we could add a sentence explain why user could need this manual step.

Reply:
Good point!